orcas/l1l2.go

```diff
@@ -691,25 +693,30 @@ func (l *L1L2Orca) Gat(req common.GATRequest) error {
 		// not be using which is then async TTL'd out. I am explicitly
 		// discounting the concurrent delete situation here and accepting that
 		// they might not be exactly correct.
-		setreq := common.SetRequest{
+		touchreq := common.TouchRequest{
 			Key:     req.Key,
 			Exptime: req.Exptime,
-			Flags:   res.Flags,
-			Data:    res.Data,
 		}
 
-		metrics.IncCounter(MetricCmdGatSetL2)
-		err := l.l2.Set(setreq)
+		metrics.IncCounter(MetricCmdGatTouchL2)
+		err := l.l2.Touch(touchreq)
 
 		if err != nil {
-			// set error? Return it as our error. Yes, the GAT succeeded in L1
-			// but if L2 didn't get the set, then likely something is seriously
-			// wrong.
-			metrics.IncCounter(MetricCmdGatSetErrorsL2)
-			metrics.IncCounter(MetricCmdGatErrors)
+			if err == common.ErrKeyNotFound {
+				// this is a problem. L1 had the item but L2 doesn't. To avoid an
+				// inconsistent view, return the same ErrNotFound and fail the op.
+				metrics.IncCounter(MetricCmdGatTouchMissesL2)
+				metrics.IncCounter(MetricCmdGatMisses)
```

Comment:
Can you add a metric for inconsistency detected?  Then we get setup an alert on it to know if it's happening or not.  The standard metrics that you've already added may be more difficult to notice if it only happens occasionally.